Automattic/wp-calypso
# FSE - add template selector access through document sidebar

#### Changes proposed in this Pull Request

* Adds access to page template selector in document sidebar.
* Allows access to template selector for already existing pages.


#### Testing instructions


Tested both locally and on sandbox.
* Run this PR.
* Navigate to add a new page.
* Verify template selector works as expected, select a template.
* Navigate to the document sidebar and the `Page Layout` dropdown.
* Verify the current template is rendered in the preview.
* Select the `change layout` button
* Verify warning modal appears and can be cancelled.
* Select 'change layout' from the warning modal.
* Select a new template and apply.
* Verify the new template is in place.

* Repeat this testing process for an already existing page.

![Screen Shot 2019-11-11 at 8 05 30 PM](https://user-images.githubusercontent.com/28742426/68633130-b4b0bb80-04be-11ea-91ce-61ceb53deb77.png)

![Screen Shot 2019-11-11 at 8 05 37 PM](https://user-images.githubusercontent.com/28742426/68633133-b8444280-04be-11ea-9091-cce10bee1fc2.png)

![Screen Shot 2019-11-11 at 8 05 56 PM](https://user-images.githubusercontent.com/28742426/68633134-ba0e0600-04be-11ea-825a-a8f0517aeb9a.png)


Fixes #37281
